src/WebformCivicrmPostProcess.php

```diff
@@ -2160,6 +2164,7 @@ private function submitIPNPayment() {
       $params['cancelURL'] = $this->getIpnRedirectUrl('cancel');
     }
 
+    $params['is_email_receipt'] = $isEmailReceipt;
```

Comment:
Can we simply use 

    $params['is_email_receipt'] = wf_crm_aval($this->data, "receipt:number_number_of_receipt", FALSE);

instead of sending `$isEmailReceipt` as a param argument?

Reply:
Sure! Like this? https://github.com/colemanw/webform_civicrm/pull/987